core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -1599,6 +1599,17 @@ private void checkHavingAliasSameAsColumn(boolean upperAlias) {
     sql(query).withMssql().ok(expected);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4690">[CALCITE-4690]
+   * Error when when executing query with CHARACTER SET in Redshift</a>. */
```

Comment:
code annotation has repeated when word？ “Error when exe.....”

Reply:
Good catch, thanks for spotting this.